engine-rest/engine-rest-openapi/src/main/templates/paths/signal/post.ftl

```diff
@@ -0,0 +1,54 @@
+{
+
+  <@lib.endpointInfo
+      id = "throwSignal"
+      tag = "Signal"
+      desc = "A signal is an event of global scope (broadcast semantics) and is delivered to all active handlers. 
+              Internally this maps to the engine's signal event received builder method RuntimeService#createSignalEvent().
+              For more information about the signal behavior, see the [Signal Events](${docsUrl}/reference/bpmn20/events/signal-events/)
+              section of the [BPMN 2.0 Implementation Reference](${docsUrl}/reference/bpmn20/)." />
+
+  "parameters" : [],
+
+  <@lib.requestBody
+      mediaType = "application/json"
+      dto = "SignalDto"
+      examples = ['"examle-1": {
+                     "summary": "POST /signal",
+                     "description": "The content of the Request Body",
+                     "value": {
+                         "name": "policy_conditions_changed",
+                          "variables": {
+                            "newTimePeriodInMonth": {
+                              "value": 24
+                           }
+                         }
+                       }
+                     }']/>
+
+  "responses" : {
+
+    <@lib.response
+        code = "204"
+        desc = "Request successful."/>
+
+    <@lib.response
+        code = "400"
+        desc = "Returned if: 
+
+                no name was given
+
+                or if the variable value or type is invalid, for example if the value could not be parsed to an integer value or the passed variable type is not supported
+
+                or if a tenant id and an execution id is specified."/>
```

Comment:
Example:
https://github.com/camunda/camunda-bpm-platform/blob/46484a1892dacb4b7dc877ee1be8abfa62f072eb/engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/modification/ProcessInstanceModificationInstructionDto.ftl#L9-L14
```suggestion
        desc = "Returned if: 

                * no name was given
                * or if the variable value or type is invalid, for example if the value could not be parsed to an integer value or the passed variable type is not supported
                * or if a tenant id and an execution id is specified."/>
```

Reply:
Thanks, I already committed the same change before I saw your suggestion.